.travis.yml

```diff
@@ -30,13 +30,20 @@ jobs:
           script:
               - vendor/bin/phpcs -p
 
+stages:
+    - Test
+    - name: Code Quality
+      if: type = pull_request
+
 allow_failures:
     - php: nightly
 
 cache:
     directories:
         - $HOME/.composer/cache/files
 
-branches:
-    only:
-        - master
+if: |
+    branch = master OR \
+    branch =~ /^(?:[0-9]|[1-9][0-9]*)\.(?:[0-9]|[1-9][0-9]*)$/ OR \
```

Comment:
Version branches are probably a necessary evil for minor versions for releasing open source packages. The more complex this is the more we should start to extract it from the single repository into a template.